openbas-api/src/main/java/io/openbas/rest/inject/service/ExecutableInjectService.java

```diff
@@ -73,6 +76,23 @@ public static String replaceCmdVariables(String cmd) {
     return result.toString();
   }
 
+  private String obfuscateCommandBase64(String command, String executor) {
```

Comment:
**question**
Can we have an Obfuscator utility ?
It would be simplier to test.